ethereum/contracts/InboundChannel.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.6;
+pragma experimental ABIEncoderV2;
+
+abstract contract InboundChannel {
+
+    uint64 public nonce;
+
+    struct Message {
+        address target;
```

Comment:
```suggestion
        address destination;
```

Reply:
I'll probably leave this low-hanging fruit for another PR. If we go ahead with this, for consistency we'll need to rename the corresponding `source` to `origin`, and then a search/replace gets a bit more involved.